datajoint/external.py

```diff
@@ -347,6 +347,8 @@ def delete(self, *, delete_external_files=None, limit=None, display_progress=Tru
                         self._remove_external_file(external_path)
```

Comment:
Why is it better to remove the external file before deleting from the external table. What if the delete from the external table fails?